--- a/input/drivers_joypad/sdl_miyoomini_joypad.c	2025-11-14 14:24:58.078522322 +0000
+++ b/input/drivers_joypad/sdl_miyoomini_joypad.c	2025-11-14 14:24:58.470525270 +0000
@@ -140,6 +140,7 @@
    input_autoconfigure_connect(
          sdl_miyoomini_joypad_name(0), /* name */
          NULL,                         /* display_name */
+         NULL,                         /* physical location */
          sdl_dingux_joypad.ident,      /* driver */
          0,                            /* port */
          0,                            /* vid */
